app/services/foreman_templates/template_exporter.rb

```diff
@@ -42,15 +44,15 @@ def export_to_git
       if new_repo || status.added.any? || status.changed.any? || status.deleted.any? || status.untracked.any?
         git_repo.commit "Templates export made by Foreman user #{foreman_git_user}"
         git_repo.push 'origin', branch
-        @export_result.exported = true
       else
-        @export_result.warning = 'No change detected, skipping the commit and push'
+        @warning = 'No change detected, skipping the commit and push'
+        @result_lines.each { |r| r.generic_info(@warning) }
```

Comment:
I wonder if we need to add the same message to each template when it is caused by the export as a whole.

Reply:
I think there is no reason for this currently, removed.